src/utils/telemetry.ts

```diff
@@ -2,13 +2,18 @@ import {ipcRenderer} from 'electron';
 
 import Nucleus from 'nucleus-nodejs';
 
+import {ValidationIntegrationId} from '@components/organisms/ValidationDrawer/integrations';
```

Comment:
here we might have the problem with importing `.tsx` to main process?

Reply:
Hmm you are correct. Damnit, I just fixed that in my other branch. Really annoying limitation. What do you think if I add jsx to TS compiler and we only use `import type`? I think it's unlikely anyone will use React components in Node process.

Reply:
Actually this is just a `.ts` file so it works properly. We can reconsider this if we actually run into it again but for this PR it is fine.